### PR TITLE
Clarify the stability of cw-storage-plus, no longer Experimental

### PR DESCRIPTION
**Pull request by ethanfrey**
Update the readme and description

### CODE REVIEW

File: packages/storage-plus/README.md [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-This has been heavily used in many production-quality contracts and
-heavily refined. The code has demonstrated itself to be stable and powerful.
-Please feel free to use it in your contracts.
+As of `cw-storage-plus` `v0.12` the API should be quite stable.
+There are no major API breaking issues pending, and all API changes will be documented
+in [`MIGRATING.md`](../../MIGRATING.md).
+
+This has been heavily used in many production-quality contracts.
+The code has demonstrated itself to be stable and powerful.
+It has not been audited, and Confio assumes no liability, butwe consider it mature enough
```
Comment (ueco-jb) [inline packages/storage-plus/README.md:21]:
```suggestion
It has not been audited, and Confio assumes no liability, but we consider it mature enough
```

**Review by ueco-jb — APPROVED** [review]
LGTM
Also, is this still beta though?
> The code has demonstrated itself to be stable and powerful. It has not been audited (...)

File: packages/storage-plus/README.md [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+to be the standard storage layer for your contracts.
```
Comment (maurolacy) [inline packages/storage-plus/README.md:22]: I would highlight this in bold:
```suggestion
to be the **standard storage layer** for your contracts.
```

**Review by maurolacy — APPROVED** [review]

File: packages/storage-plus/README.md (hunk 1 above)
Comment (ethanfrey) [reply packages/storage-plus/README.md:21]: I would consider anything beta until it is 1.0.
No longer alpha or experimental though.

**Pull request merged**